dashboard/test/ui/runner.rb

```diff
@@ -353,8 +351,11 @@ def configure_for_eyes
   ENV['BATCH_ID'] = "#{GIT_BRANCH}_#{SecureRandom.uuid}".gsub(/[^\w-]+/, '_')
 
   # Also seed eyes data
-  puts "Seeding eyes data"
-  UiTestHelpers.seed_pd_eyes_data
+  ChatClient.log "Seeding eyes data"
+  unless system('ruby ../../../bin/test_utils/seed_pd_eyes_data.rb ')
```

Comment:
nit: I'd prefer an absolute path here, ideally something like `CDO.bin_dir('test_utils/seed_pd_eyes_data.rb')`. Note [bin_dir](https://github.com/code-dot-org/code-dot-org/blob/5d389d36456c5b7ead3e4d6d5fa8657f46d30b5d/deployment.rb#L369-L407) would need to be added here